wordpress-mobile/WordPress-Android
# Issue/6855 autofill email list

### Fix
Disable the [autofill feature](https://developer.android.com/guide/topics/text/autofill.html) for the login email and email/password screens to avoid the persistent overlay bug as described in https://github.com/wordpress-mobile/WordPress-Android/issues/6855.  Note that these changes include updating the compile version, updating the support library dependencies, and adding a repository in the `build.gradle` file, which are required to add the `importantForAutofill` attribute.

### Test
0. Clear app data.
1. Tap ***Log In*** button.
2. Notice autofill is not shown for ***Email address*** field.
3. Enter email address manually in ***Email address*** field.
4. Tap ***Next*** button.
5. Tap ***Enter your password instead*** button.
6. Notice autofill is not shown for ***Password*** field.

#### Review
@hypest and @mzorz, can you review this one since you were able to trigger autofill?
